src/main/java/bizbook/model/AddressBook.java

```diff
@@ -54,6 +58,8 @@ public void setPersons(List<Person> persons) {
     public void resetData(ReadOnlyAddressBook newData) {
         requireNonNull(newData);
 
+        // Empty the pinned person list since we are reseting the addressbook
+        setPinnedPersons(new ArrayList<Person>());
```

Comment:
Can be removed by the above change.

Reply:
We still need this line since when we `clear` the address book it will make a empty address book and set the list